openprescribing/bq_public_tables/practice.sql

```diff
@@ -0,0 +1,24 @@
+SELECT
+    code AS id,
+    name,
+    ccg_id,
+    pcn_id,
+    status_code,
+    setting,
+    open_date,
+    close_date,
+    join_provider_date,
+    leave_provider_date,
+    CASE
+      WHEN COALESCE(address1, address2, address3, address4, address5) IS NULL THEN NULL
+      ELSE CONCAT(
+        address1,
+        CASE WHEN address2 IS NULL THEN '' ELSE CONCAT(', ', address2) END,
+        CASE WHEN address3 IS NULL THEN '' ELSE CONCAT(', ', address3) END,
+        CASE WHEN address4 IS NULL THEN '' ELSE CONCAT(', ', address4) END,
+        CASE WHEN address5 IS NULL THEN '' ELSE CONCAT(', ', address5) END
```

Comment:
Why are we not leaving the fields as-is?

One reason this concat might not be good is if someone wanted to print envelopes (for example): the separate fields presumably denote sensible line breaks, whereas there will be commas within lines in the data.

Alternatively we could concat with line breaks

Reply:
Good question, not sure.  I'll remove it.